tests/node_converters/pad_test.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
+import onnx
+import pytest
+
+from tests.utils.common import check_onnx_model
+from tests.utils.common import make_model_from_nodes
+
+
+def _test_pad(
+        input_array: np.ndarray,
+        opset_version: int,
+        input_pads: np.ndarray = None,
+        **kwargs,
+) -> None:
+
+    test_inputs = {
+        'x': input_array,
+    }
+
+    if opset_version >= 11:
+        test_inputs['pads'] = input_pads
+
+    node = onnx.helper.make_node(
+        'Pad',
+        inputs=list(test_inputs),
+        outputs=['y'],
+        **kwargs,
+    )
+
+    model = make_model_from_nodes(
+        nodes=node,
+        initializers={},
+        inputs_example=test_inputs,
+        opset_version=opset_version,
+    )
+    check_onnx_model(model, test_inputs)
+
+
+@pytest.mark.parametrize(
```

Comment:
('pads,opset,mode')

Reply:
I don't parameterize opset version, because Pad node has different types of pads attribute in different opset versions.
For example, opset version = 13 needs pads with torch.Tensor type, opset_version=2 needs List of pads.

Reply:
You can say that number of strings does not change. Yes, but when you add new opset version you just add a number to tuple, instead a line.
```python
if opset in (11, 13):
    pads = np.array(pads, dtype=np.int64)
_test_pad(input_array=input_tensor, mode=mode, opset_version=opset, input_pads=pads)
```